src/index.js

```diff
@@ -20,10 +20,98 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+const operators = ['+', '-', '*', '/', '='];
+
+function handleCalculator(orgnValue, windowNumber, numOrOperator) {
+  // https://stackoverflow.com/questions/20169217/how-to-write-isnumber-in-javascript
+  // _isNumber
+
+  console.log('orgnValue: ', orgnValue, ', windowNumber: ', windowNumber , ', numOrOperator: ', numOrOperator);
```

Comment:
```suggestion
  console.log({ orgnValue,  windowNumber, numOrOperator });
```
위와 같은방법으로 콘솔을 찍으면, 같은 결과를 더 쉽게 도출해낼 수 있을거에요!